dashboard/config/scripts/levels/U3L03 - draw step.level

```diff
@@ -66,9 +66,7 @@
     "encrypted_examples": "zkDI0HDKHo5c8J74y03IKljJJN88FpvWCVjolNR18sIy18c5dx0tu25snUyX\nRtR7\n",
     "lock_zero_param_functions": "true",
     "execute_palette_apis_only": "true",
-    "contained_level_names": [
-      "Draw Diamond"
-    ],
+    "contained_level_names": null,
```

Comment:
Yeah, these levels don't exist. `Level#contained_levels` used to silently ignore them, but I think it's better to have it complain loudly.